client/web/src/site-admin/SiteAdminRepositoriesPage.tsx

```diff
@@ -224,6 +238,20 @@ export const SiteAdminRepositoriesPage: React.FunctionComponent<React.PropsWithC
                 </Link>
                 .
             </Text>
+            {licenseInfo != null && (licenseInfo.codeScaleCloseToLimit || licenseInfo.codeScaleExceededLimit) && (
+                <Alert variant={licenseInfo.codeScaleExceededLimit ? 'danger' : 'warning'}>
+                    <H4>
+                        {licenseInfo.codeScaleExceededLimit ? (
+                            <>You've used all 100GB of storage</>
```

Comment:
```suggestion
                            <>You've used all 100GiB of storage</>
```

Just to be consistent with the site alerts?

Reply:
Yeah makes sense, good spot!